docs/usage-guide/topics/ch15-post-quantum.md

```diff
@@ -1,16 +1,28 @@
+s2n-tls supports both post-quantum key exchange and post-quantum authentication for TLS1.3.
 
-Specifically, s2n-tls supports hybrid key exchange. PQ hybrid key exchange involves performing both classic ECDH key exchange and post-quantum Kyber key exchange, then combining the two resultant secrets. This strategy combines the high assurance of the classical key exchange algorithms with the quantum-resistance of the new post-quantum key exchange algorithms. If one of the two algorithms is compromised, either because advances in quantum computing make the classic algorithms insecure or because cryptographers find a flaw in the relatively new post-quantum algorithms, the secret is still secure. Hybrid post-quantum key exchange is more secure than standard key exchange, but is slower and requires more processing and more network bandwidth.
+## Key Exchange: ML-KEM / Kyber
+
+Currently, only [ML-KEM](https://csrc.nist.gov/pubs/fips/203) / [Kyber](https://pq-crystals.org/kyber/) are supported for post-quantum key exchange. "ML-KEM" is the name given to the NIST standardized version of Kyber.
+
+Specifically, s2n-tls supports hybrid key exchange. PQ hybrid key exchange involves performing both classic ECDH key exchange and post-quantum key exchange, then combining the two resultant secrets. This strategy combines the high assurance of the classical key exchange algorithms with the quantum-resistance of the new post-quantum key exchange algorithms. If one of the two algorithms is compromised, either because advances in quantum computing make the classic algorithms insecure or because cryptographers find a flaw in the relatively new post-quantum algorithms, the secret is still secure. Hybrid post-quantum key exchange is more secure than standard key exchange, but is slower and requires more processing and more network bandwidth.
 
 Careful: An s2n-tls server that enables post-quantum cryptography will mandate post-quantum key exchange with any client advertising post-quantum algorithms. This can result in a retry and an extra round trip if the client does not initially send a post-quantum key share. The rational behind this behavior is that post-quantum users prioritize security over the potential cost of an extra round trip.
 
+## Authentication: ML-DSA
+
+Currently, only [ML-DSA](https://csrc.nist.gov/pubs/fips/204) is supported for post-quantum authentication.
+
+In order to use ML-DSA, you must configure s2n-tls to use an ML-DSA certificate, just as you would configure an RSA or ECDSA certificate. See [certificates](./ch09-certificates.md).
+
+Note: s2n-tls may not successfully parse ML-DSA private keys generated by Openssl. There are three possible formats defined for an ML-DSA private key: "seed", "expanded", and "both". Openssl currently generates keys in the "both" format by default, but AWS-LC is unable to parse keys in the "both" format. AWS-LC will accept either the "seed" or "expanded" formats. See https://github.com/aws/aws-lc/issues/2413.
```

Comment:
It's not clear immediately clear what the relevance of this problem is for s2n-tls. Where would an error occur? When loading certificates and the underlying libcrypto you're using is awslc?

Reply:
I just removed the paragraph. aws-lc has a PR out and approved to fix it, so we'd need to remove/modify this warning basically immediately anyway: https://github.com/aws/aws-lc/pull/2416